pkg/controller/sub_controller/disaggregated_cluster/computegroups/util.go

```diff
@@ -17,8 +17,36 @@
 
 package computegroups
 
+import (
+	dv1 "github.com/apache/doris-operator/api/disaggregated/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
 // regex
 var (
 	compute_group_name_regex = "[a-zA-Z](_?[0-9a-zA-Z])*"
 	compute_group_id_regex   = "[a-zA-Z](_?[0-9a-zA-Z])*"
 )
+
+func ownerReference2ddc(obj client.Object, cluster *dv1.DorisDisaggregatedCluster) bool {
```

Comment:
ownerReferenceBelongToDDC

Reply:
Ownerreference presents the resource depended on other. the function find the resource owner reference to ddc.